content/webapp/pages/books/[bookId].tsx

```diff
@@ -52,8 +53,26 @@ type MetadataProps = {
 
 const Metadata: FunctionComponent<MetadataProps> = ({ label, value }) => (
   <>
-    <MetadataKey>{label}</MetadataKey>
-    <MetadataValue>{value}</MetadataValue>
+    <MetadataKey
+      $sizeMap={{
+        s: [4],
+        m: [4],
+        l: [4],
+        xl: [4],
+      }}
+    >
+      {label}
+    </MetadataKey>
+    <MetadataValue
+      $sizeMap={{
+        s: [8],
+        m: [8],
+        l: [8],
+        xl: [8],
+      }}
```

Comment:
Could move these to the SC? I don't know. What do you think?

Reply:
TypeScript complains if they're not attributes on the implementation. I'm not sure if there's a way to get around that?

Reply:
Oh strange, ok, maybe too many layers... 🤷‍♀️